client/tests/RequestTerm.integ.test.js

```diff
@@ -56,13 +51,7 @@ it('Renders form and submit term', async () => {
 
   // Act
   fireEvent.click(snackbarDismissBttn)
-  try {
-    await waitForElement(() => getByText(/Your request for Queer was successful/))
-  } catch (e) {
-    // Assert
-    expect(e.message).toBe('Unable to find the element')
-  }
 
   // Assert
-  expect(requestForm).not.toContainElement(snackbar)
+  expect(queryByText(/Your request for Queer was successful/)).not.toBeInTheDocument()
```

Comment:
`queryBy...` returns null instead of throwing like `getBy...`

Timing doesn't seem to matter, if you comment out the click the test will fail

Reply:
yeah, I actually tried to test waiting for the snackbar to disappear, but no luck with that, and then I decided to test the other method which is the user 'removing' the snackbar by clicking the dismiss button. I think I can create a new integration test just to wait for the element to disappear but I'm happy with testing the dismiss button too. As of now the time it takes to remove the snackbar is 3 seconds btw :)

Reply:
Another think I'm thinking is that if we want to, I can remove the customized query and just let the code: expect(requestForm).not.toContainElement(snackbar)
I like that because I think it reads very clear, we are expecting the form not containing the snackbar anymore, since we clicked the dismiss button. What do you think?